src/components/ReplyComment/index.js

```diff
@@ -47,6 +47,7 @@ const ReplyCommentId = ({
     updateFeed,
     scrollViewRef,
     createComment,
+    onSaveHeight,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSaveHeight` function is added to the destructuring assignment from `useReplyComment`. Ensure that this function is properly defined and exported in the `useReplyComment` hook.
```diff
-     getThisComment
+     onSaveHeight,
+     getThisComment
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->